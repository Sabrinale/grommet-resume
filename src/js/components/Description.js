import React from 'react'

import { Paragraph } from 'grommet'

export const Description = ({ children }) => (
  <Paragraph margin={{ top: '0px', left: 'xsmall' }}>{children}</Paragraph>
)
