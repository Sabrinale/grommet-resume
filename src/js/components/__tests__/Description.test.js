import React from 'react'
import renderer from 'react-test-renderer'

import { Description } from '../Description'

test('renders description', () => {
  const component = renderer.create(<Description>test</Description>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
