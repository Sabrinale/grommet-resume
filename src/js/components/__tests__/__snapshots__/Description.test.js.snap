// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders description 1`] = `
<p
  className="StyledParagraph-tbetod-0 jzEULC"
>
  test
</p>
`;
